src/components/toggle-button/assistant-departure-toggle-button.tsx

```diff
@@ -0,0 +1,80 @@
+import Link from 'next/link';
+import style from './assistant-departure-toggle-button.module.css';
+import { PageText, TranslatedString, useTranslation } from '@atb/translations';
+import { useState } from 'react';
+import { Typo } from '../typography';
+import { AnimatePresence, motion } from 'framer-motion';
+
+type AssistantDepartureTogglProp = {
+  href?: string;
+};
+
+type LinkProps = {
+  name: string;
+  title: TranslatedString;
+  link: string;
+};
+
+const links: LinkProps[] = [
+  {
+    name: 'assistant',
+    title: PageText.Assistant.title,
+    link: '/assistant',
+  },
+  {
+    name: 'departures',
+    title: PageText.Assistant.search.buttons.find.title,
+    link: '/departures',
+  },
+];
+
+const setInitialActiveLink = (href: string | undefined) => {
+  return href ? links[1].name : links[0].name;
+};
+
+const handleClick = (linkName: string, setActiveLink: Function) => {
+  setActiveLink(linkName);
+};
+
+const AssistantDepartureToggle = ({ href }: AssistantDepartureTogglProp) => {
+  const { t } = useTranslation();
+  const [activeLink, setActiveLink] = useState(() =>
+    setInitialActiveLink(href),
+  );
+
+  return (
+    <div className={style.container}>
+      {links.map((link, index) => (
+        <Link
+          key={index}
```

Comment:
Indexes are generally not the best to use for keys. @mikaelbr probably has a blog post to link.

Reply:
Can you send me the link?

Reply:
create an image of a generic male figure with a strong physique, wearing a Batman costume

Reply:
This is usually a very good point and something to keep in mind every time `key` is used. I think however, for this particular case, this won't be an issue as the object is static and thus the index will never update.